app/src/main/res/layout/location_row.xml

```diff
@@ -21,7 +22,7 @@
         android:id="@+id/txtName"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="TextView"
+        android:text="@string/textview"
```

Comment:
strings.xml is the best place for string literals. You can have a different strings.xml for each language you support, and the Android operating system will automatically choose the correct file, based on the user's preferred language.